pentestportal_ec2.tf

```
@@ -80,7 +80,8 @@ resource "aws_eip" "pentestportal" {
   provider = aws.provisionassessment
 
   tags = {
-    Name             = format("PentestPortal%d EIP", count.index + 1)
+    Name = format("PentestPortal%d EIP", count.index + 1)
+    # This instance type is always excluded from egress IP publishing
     "Publish Egress" = "False"
```

Based on https://github.com/cisagov/cool-assessment-terraform/pull/240#discussion_r1600570216 I think this should be changed.

```suggestion
    "Publish Egress" = "false"
```

---

Anything other than `true` won't actually matter, but I still agree with you. 😄